docs/application/native/get-started/wearable/first-app.md

```diff
@@ -284,51 +284,54 @@ For more information on using the emulator features, see [Using Emulator Control
 To run the application on a target device:
 
 1.  Connect the wearable target device to your computer:
-    1.  Define settings on the device:
-        -   Go to **Settings \> Connections**, and switch on Bluetooth.
 
-            ![Switch on Bluetooth](media/emulator_target_bt.png)
+    1. Define settings on the device:
 
-            ![Switch on Bluetooth](media/emulator_target_bt2.png)
+        -   Execute the **Settings** application.
 
-        -   Go to **Settings \> Connections**, and switch on Wi-Fi.
+            ![Settings](media/wearable_settings_app.png)
+
+        -   Go to **Connections > Wi-Fi**, and switch on Wi-Fi.
+
+            ![Enable Wi-Fi](media/testing_your_app_on_gear3.png) ![Select Wi-Fi AP](media/testing_your_app_on_gear4.png)
+
+        -   Go to **Wi-Fi networks > Scan**.
+
+            ![Enable Wi-Fi](media/testing_your_app_on_gear5.png) ![Select Wi-Fi AP](media/testing_your_app_on_gear6.png)
```

Comment:
Just a question. 
Is the first image testing_your_app_on_gear5.png comes before testing_your_app_on_gear6.png?
My question is because, it looks like the left image is the result of the right image. 
(Scan in Wi-Fi networks, and the result is TizenCSAPI connected.)

![image](https://user-images.githubusercontent.com/9958374/68451103-caa94c80-0230-11ea-9551-6b0b5f9dd4c5.png)